examples/rap_blocksworld/inference.py

```diff
@@ -86,7 +86,7 @@ def rap_bw(base_model: LanguageModel,
     import random
     import torch
     import torch.backends.cudnn
-    from reasoners.lm import LLaMAModel
+    from reasoners.lm import LLaMAModel, LlamaModel
```

Comment:
Very confusing

Reply:
I just want to align with meta and hf's naming